php/src/AbstractJws.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
```

Comment:
The strict type declaration is needed at the top of each file.
```suggestion
declare(strict_types=1);

```

If you don't have it then types will be coerced. (At least on PHP 7.4, I'm pretty sure the same will happen on 8).